runtime/worker.rs

```diff
@@ -480,7 +480,10 @@ impl MainWorker {
     &mut self,
     wait_for_inspector: bool,
   ) -> Result<(), AnyError> {
-    self.js_runtime.run_event_loop(wait_for_inspector).await
+    tokio::task::unconstrained(
+      self.js_runtime.run_event_loop(wait_for_inspector),
+    )
```

Comment:
We should apply that to the top-level future in `main.rs` instead - this is done to ensure that the main future is never pre-empted because it exceeds the coop budget.